tsrc/cli/main.py

```diff
@@ -48,7 +48,7 @@ def die(message):
 
 def workspace_subparser(subparser, name):
     parser = subparser.add_parser(name)
-    parser.add_argument("-w", "--workspace", dest="workspace_path")
+    parser.add_argument("--workspace", dest="workspace_path")
```

Comment:
Why did you get rid of the `-w` short option ?

Never mind, it's used for `-w,--watch`. Fair enough :)

Reply:
added a comment :)